package.json

```diff
@@ -70,6 +70,7 @@
     "redux-actions": "^2.6.5",
     "redux-immutable-state-invariant": "^2.1.0",
     "redux-logger": "^3.0.6",
+    "redux-mock-store": "^1.5.4",
```

Comment:
I think this should be a dev dependency no?

Reply:
You're right! Good catch!